.pre-commit-config.yaml

```diff
@@ -72,5 +72,7 @@ repos:
       - id: validate-helm-charts
         name: validate helm charts with kubeconform
         entry: bash scripts/validate-helm.sh
-        language: system
+        language: python
         pass_filenames: false
+        additional_dependencies:
+          - PyYAML==6.0.1
```

Comment:
Should we update the version here too? https://github.com/ray-project/kuberay/blob/0821fd3d6bf63d1b5dd78fcb7b337c959302f2d5/scripts/requirements.txt#L1